niftynet/layer/loss_classification_multi.py

```diff
@@ -0,0 +1,265 @@
+# -*- coding: utf-8 -*-
+"""
+Loss functions for multi-class classification
+"""
+from __future__ import absolute_import, print_function, division
+
+import numpy as np
+import tensorflow as tf
+
+from niftynet.engine.application_factory import LossClassificationMultiFactory
+from niftynet.layer.base_layer import Layer
+
+
+class LossFunction(Layer):
+    def __init__(self,
+                 n_class,
+                 n_rater,
+                 loss_type='CrossEntropy',
+                 loss_func_params=None,
+                 name='loss_function'):
+
+        super(LossFunction, self).__init__(name=name)
+        self._num_classes = n_class
+        self._num_raters = n_rater
+        if loss_func_params is not None:
+            self._loss_func_params = loss_func_params
+        else:
+            self._loss_func_params = {}
+        self._data_loss_func = None
+        self.make_callable_loss_func(loss_type)
+
+    def make_callable_loss_func(self, type_str):
+        self._data_loss_func = LossClassificationMultiFactory.create(type_str)
+
+    def layer_op(self,
+                 pred_ave=None,
+                 pred_multi=None,
+                 ground_truth=None,
+                 weight_batch=None,
+                 var_scope=None, ):
+        """
+        Compute loss from `prediction` and `ground truth`,
+
+        if `prediction `is list of tensors, each element of the list
+        will be compared against `ground_truth`.
+
+        :param prediction: input will be reshaped into (N, num_classes)
```

Comment:
add a few lines of doc for the expected shape of param `pred_multi`and `pred_ave`?

Reply:
Done!